sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClientOptions.cs

```diff
@@ -35,10 +36,14 @@ public CommunicationIdentityClientOptions(ServiceVersion version = LatestVersion
         /// </summary>
         public enum ServiceVersion
         {
+#pragma warning disable CA1707 // Identifiers should not contain underscores
+            /// <summary>
+            /// The V2021_02_22_preview1 of the identity service.
+            /// </summary>
+            V2021_02_22_preview1 = 0,
```

Comment:
The version should be higher than 2021_03_07 both in terms of date and value

Reply:
@RezaJooyandeh - Are you saying the client has to support a different version (which is higher than the stable version V2021_03_07)? This preview version is the only one supported by TURN service currently. I'm not sure if this can be changed on the client side, without changing the service side?

Reply:
Or are you suggesting to only update the enum, but not the actual version string?

Reply:
We should be updating the version in both the client and service side so that it is higher than the GA API version. Right now it is 2021-02-22-preview1 which is lower than the GA version 2021-03-07.

Reply:
The way I understand it, there's 2 versions; a SDK version and a target Service version. The SDK version would change for things that only affect the SDK (like a method parameter rename, bug fix or refactor to use a common REST client) and the target service version would change for service behavior changes. 

As far as I'm seeing this is only a SDK change - unless there's more changes in this PR than just an SDK refactor that I'm not seeing?

It seems strange to me that we'd need to increase the service version and deploy this service without making any changes to the service?

Reply:
Resolved offline to leave this change as is (reason being this private preview version will be overridden by upcoming public preview version very soon, so not worth making the service version changes now)